source

```diff
@@ -100158,6 +100158,15 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
      rules don't apply any additional restrictions, make their way into the DOM.</p>
 
     <tr>
+     <td><dfn data-x="parse-error-non-conforming-doctype">non-conforming-doctype</dfn>
+     <td><p>This error occurs if the parser encounters the <span
+     data-x="syntax-doctype">DOCTYPE</span> that is correctly placed as a document preamble and that
```

Comment:
data-x="syntax-doctype">DOCTYPE</span> ~that is~ correctly placed as a document preamble and ~that~